File: sandbox/node/src/TeamRemoveMemberEmailAddress.ts
import * as fs from 'fs';
import api from "@dropbox/sign"
import models from "@dropbox/sign"

const apiCaller = new api.TeamApi();
apiCaller.username = "YOUR_API_KEY";
// apiCaller.accessToken = "YOUR_ACCESS_TOKEN";

const teamRemoveMemberRequest = new models.TeamRemoveMemberRequest();
teamRemoveMemberRequest.emailAddress = "teammate@dropboxsign.com";
teamRemoveMemberRequest.newOwnerEmailAddress = "new_teammate@dropboxsign.com";

apiCaller.teamRemoveMember(
    teamRemoveMemberRequest,
).then(response => {
  console.log(response.body);
}).catch(error => {
  console.log("Exception when calling Team#teamRemoveMember:");
  console.log(error.body);
});

File: sandbox/python/src/TeamRemoveMemberEmailAddress.py
from datetime import date, datetime
from pprint import pprint

from dropbox_sign import ApiClient, ApiException, Configuration, api, models

configuration = Configuration()

with ApiClient(configuration) as api_client:
    team_remove_member_request = models.TeamRemoveMemberRequest(
        email_address="teammate@dropboxsign.com",
        new_owner_email_address="new_teammate@dropboxsign.com",
    )

    try:
        response = api.TeamApi(api_client).team_remove_member(
            team_remove_member_request=team_remove_member_request,
        )

        pprint(response)
    except ApiException as e:
        print("Exception when calling Team#team_remove_member: %s\n" % e)
